infra/blueprint-test/pkg/gcloud/gcloud.go

```diff
@@ -107,6 +117,18 @@ func Run(t testing.TB, cmd string, opts ...cmdOption) gjson.Result {
 	return gjson.Parse(op)
 }
 
+// TerraformVet executes gcloud beta terraform vet
+func TerraformVet(t testing.TB, planFilePath string, policyLibraryPath string) gjson.Result {
```

Comment:
nit: slightly smaller name which I think conveys the intent.
```suggestion
func TFVet(t testing.TB, planFilePath, policyLibraryPath string) gjson.Result {
```

Reply:
fixed